coursedata/adventures/de.yaml

```diff
@@ -603,164 +609,195 @@ adventures:
+                    ## Lass uns zeichnen
+                    In Level 5 kannst Du mit dem Befehl `if` Entscheidungen treffen.
+                    Diú kannst Dich zum Beispiel zwischen verschiedenen Formen entscheiden.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120
-                    else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
+                    if Form is Dreieck Winkel is 120
+                    else Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120 else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
+                    if Form is Dreieck Winkel is 120 else Winkel is 90
+                    turn Winkel
                     forward 25
             8:
                 story_text: |
-                    ## Let's draw
-                    Now that we can repeat several lines, we can make figures more easily.
-                    We only have to set the angle once and then use that variable in the `repeat`.
+                    ## Lass uns zeichnen
+                    Jetzt, da wir einzelne Linien wiederholen können, fällt uns das Zeichnen einzelner Formen leichter.
+                    Wir setzen den Winkel einmalig und nutzen dann die Variable mit `repeat`.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    angle = 90
+                    Winkel = 90
                     repeat 10 times
-                        turn angle
+                        turn Winkel
                         forward 50
                     ```
+                story_text_2: |
+                    ## Interaktiv
+                    Wir können auch das Programm verbessern, das verschiedene Formen zeichnet.
+                    Bekommst Du raus, wie weit die Schildkröte sich drehen muss?
+                    Vervollständige den Code und zeichne jedes Vieleck, das Du möchtest.
```

Comment:
```suggestion
                    Vervollständige den Code und Du kannst jedes Vieleck zeichnen, das Du möchtest.
```

Reply:
> Can you live with a bit of deviating from the original?

Sure. But I'm unsure whether the current deviation doesn't change the meaning too much.

"[Z]eichne jedes Vieleck, das Du möchtest." can be understood a task — and a rather daunting one at that. To keep it almost as short, how about

```suggestion
                    Vervollständige den Code und er zeichnet jedes Vieleck, das Du möchtest.
```

Reply:
I meant to let it refer to "den Code", thus "er". Assigning agency to the turtle with "sie" would work, too, as it has been mentioned in the sentence before this one.